DOCS-8952 Amazon EC2 (#1071)
mulesoft/docs-connectors: amazon-ec2/2.3/modules/ROOT/pages/amazon-ec2-connector-xml-maven.adoc

. Include the Amazon EC2 namespace and schema location in your configuration file.
+
Namespace: `+http://www.mulesoft.org/schema/mule/ec2+` +
Schema Location: `+http://www.mulesoft.org/schema/mule/ec2/current/mule-ec2.xsd+`
+
These statements appear in the `<mule>` block that starts a Mule app:
+
[source,xml,linenums]
----
<mule xmlns:ec2="http://www.mulesoft.org/schema/mule/ec2" 
	xmlns="http://www.mulesoft.org/schema/mule/core"
	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core 
	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
	http://www.mulesoft.org/schema/mule/ec2 
	http://www.mulesoft.org/schema/mule/ec2/current/mule-ec2.xsd">
----
. Create a global Amazon EC2 configuration outside your flows,
using the following global configuration code:
+
[source,xml,linenums]
----
<ec2:config name="Amazon_EC2_configuration" doc:name="Amazon EC2 Configuration" >
	<ec2:basic-connection
		accessKey="${amazon.accesskey}"
		secretKey="${amazon.secretkey}"
		region="${amazon.region}"
		sessionToken="{amazon.sessionToken}"/>
</ec2:config>
----
+

> . Include this XML snippet in your `pom.xml` file:

Reviewer: ```suggestion
To obtain the most up-to-date `pom.xml` file information:
```

> . Include this XML snippet in your `pom.xml` file:

Reviewer: ```suggestion
== Add a POM File Dependency 

The Apache Maven `pom.xml` file generated by Anypoint Studio contains dependencies for a Mule app. If you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector: 

```

+
[source,xml,linenums]
----
<dependency>
    <groupId>com.mulesoft.connectors</groupId>
    <artifactId>mule-amazon-ec2-connector</artifactId>
    <version>x.x.x</version>
    <classifier>mule-plugin</classifier>
</dependency>
----

Replace `x.x.x` with the version that corresponds to the connector you are using. 

> == Obtain POM File Information From Exchange

Reviewer: ```suggestion

```

To obtain the most up-to-date `pom.xml` file information:

. Go to https://www.mulesoft.com/exchange/[Anypoint Exchange].
. In Exchange, click *Login* and supply your Anypoint Platform username and password.
. In Exchange, search for `amazon ec2`.
. Select *Amazon EC2 Connector*.
. Click *Dependency Snippets* near the upper right of the screen.

== Next Step

After completing your namespace and `pom.xml` file, you can try 
the xref:amazon-ec2-connector-examples.adoc[Examples].

== See Also

> https://help.mulesoft.com[MuleSoft Help Center]

Reviewer: ```suggestion
* xref:connectors::introduction/introduction-to-anypoint-connectors.adoc[Introduction to Anypoint Connectors]
* https://help.mulesoft.com[MuleSoft Help Center]
```
